testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/AbstractJavascriptTest.java

```diff
@@ -214,4 +215,11 @@ public JavascriptStateValidator assertEventsContains(String text) {
     public JavascriptStateValidator assertEventsDoesntContain(String text) {
         return buildFunction(this::assertEventsWebElementDoesntContain, text);
     }
+
+    public void assertErrorResponse(WebDriver drv, Object output, WebElement evt) {
```

Comment:
This can be replaced by this to work with comments in `JavascriptAdapterTest`.

```
    public void assertErrorResponse(String expectedError, WebDriver drv, Object output, WebElement evt) {
        Assert.assertNotNull("Empty error response", output);
        Assert.assertTrue("Invalid error response type", output instanceof Map);
        assertThat((Map<String, String>) output, anyOf(hasEntry("error", expectedError), hasEntry("error_description", expectedError)));
    }
    
    public JavascriptStateValidator assertErrorResponse(String expectedError) {
        return buildFunction(this::assertErrorResponse, expectedError);
    }
```